src/d3a/d3a_core/export.py

```diff
@@ -465,6 +466,48 @@ def _plot_ess_soc_history(self, storage_list: list, subdir: str, root_name: str)
         output_file = os.path.join(plot_dir, 'ess_soc_history_{}.html'.format(root_name))
         PlotlyGraph.plot_bar_graph(barmode, title, xtitle, ytitle, data, output_file)
 
+    def _plot_stock_info_per_area_per_market_slot(self, area, plot_dir):
+        """
+        Plots stock stats for each knot in the hierarchy per market_slot
+        """
+
+        area_stats = self.endpoint_buffer.area_market_stocks_stats.state[area.name]
+        stats_plot_dir = os.path.join(plot_dir, "offer_bid_trade")
+        mkdir_from_str(stats_plot_dir)
+
+        for market_slot_date, markets in area_stats.items():
+            data = list()
+            for tick_slot, info_dict in markets.items():
+                data.append(
+                    go.Scatter(x=[tick_slot],
+                               y=[info_dict['rate']],
+                               name=info_dict['tool_tip'])
+                )
+            output_file = os.path.join(
+                stats_plot_dir, f"offer_bid_trade_history_{str(market_slot_date)}.html"
+            )
+            barmode = "relative"
+            title = f"OFFER BID TRADE AREA: {area.name} | MARKET: {str(market_slot_date)})"
+            xtitle = 'Time'
+            ytitle = 'Rate [€ cents / kWh]'
+            PlotlyGraph.plot_bar_graph(
```

Comment:
I was under the impression that Sarah wanted a line graph, or am I mistaken ?

Reply:
Also discussed with Sarah on comments in ticket, she is okay with scattered plots, though following a uniform colouring convention